### PR TITLE
Admin merchant status

### LINKED ISSUE #20: User Story 50, Admin enables a merchant account
```
As an admin merchant
When I visit the merchant index page
And I click on a "enable" button for a disabled merchant
I am returned to the admin's merchant index page
And I see a flash message that the merchant's account is now enabled
And I see that the merchant's account is now enabled
This merchant can now log in
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #25

child of #79

depends on #22

### LINKED ISSUE #21: User Story 49, Admin disables a merchant account
```
As an admin merchant
When I visit the merchant index page
And I click on a "disable" button for an enabled merchant
I am returned to the admin's merchant index page
And I see a flash message that the merchant's account is now disabled
And I see that the merchant's account is now disabled
This merchant cannot log in
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #25

child of #79

depends on #22

### PR DESCRIPTION
**Pull request by allieIsOkay**
- Adds tests for admin ability to enable and disable merchant users from '/merchants' route
- Adds logic to admin/users controller to support alternate redirect for merchants and regular users
- closes #20 
- closes #21

### CODE REVIEW

**Review by timnallen — APPROVED** [review]
Nice refactoring! DRY code is good code.

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -24,19 +24,26 @@ def update
...
+    @user = User.find(params[:id])
+    @user.update(disabled: true)
     flash[:primary] = 'You have disabled a user'
-    redirect_to admin_users_path
+    redirect_helper
   end
 
   private
 
+  def redirect_helper
```
Comment (csvlewis) [inline app/controllers/admin/users_controller.rb:42]: Nice helper method, I like how you were able to use the same code for different user types

**Review by csvlewis — APPROVED** [review]

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (teresa-m-knowles) [reply app/controllers/admin/users_controller.rb:42]: I agree!
